webapp/portlet/src/main/webapp/common/js/SuggesterService.js

```diff
@@ -38,11 +38,18 @@ function searchUsers(filter, items, typeOfRelation, searchOptions) {
     typeOfRelation: typeOfRelation || 'mention_activity_stream',
     currentUser: eXo.env.portal.userName,
   };
-  if (searchOptions) {
+
+  let params = null;
+  let url = null;
+  if (searchOptions && !searchOptions.searchUrl) {
     Object.assign(options, searchOptions);
+    params = $.param(options);
+    url = '/portal/rest/social/people/suggest.json?'.concat(params);
+  } else {
+    url = searchOptions.searchUrl.concat(filter);
   }
```

Comment:
Could we let it have other olptions like this ? WDYT ?

```suggestion
  if (searchOptions) {
    if(searchOptions.searchUrl) {
      url = searchOptions.searchUrl;
      delete searchOptions.searchUrl;
    } else {
      url = '/portal/rest/social/people/suggest.json?';
    }
    Object.assign(options, searchOptions);
    params = $.param(options);
    url = url.concat(params);
  }
```